index.html

```diff
@@ -110,16 +110,13 @@ <h2>Who are the Freetail Hackers?</h2>
 	</div>
 	<object data="assets/us.svg" type="image/svg+xml" width="600px" height="600px"></object>
 </section>
-<section id="mailing">
```

Comment:
Are we removing the entire mailing section?

Reply:
I think so?